### ISSUE TITLE
 How to use redux-actions with redux-undo?

### ISSUE THREAD

**Issue by janryWang** [acdlite/redux-actions#171]

**Comment by yangmillstheory** [comment]
Could you give a code example and state what problem you're having?

**Comment by janryWang** [comment]
Thanks, I was resolved

**Comment by laustdeleuran** [comment]
+1

@janryWang How did you end up solving this?

**Comment by laustdeleuran** [comment]
What I ended up doing was restructuring my state a little bit. We're building a painting-type application, and our state was something like this:


```
const DEFAULT_STATE = {
  activeSwatchIndex: -1,
  swatches: [],
  editor: { ... }
}
```

With all the actions for updating the editor and your swatches in the same `handleActions` call. But I only needed to make a few of the actions undoable, so I ended up extracting those out into it's own `handleActions` call, which I then wrapped in `undoable`, like this:

```
const editorReducer = undoable(handleActions({ ...reducers }));
const settingsReducer = handleActions({ ...otherReducers });
export default combineReducers({
  editor: editorReducer,
  settings: settingsReducer,
});
```

That way, only part of my state tree is wrapped in `undoable`.
